src/java.base/share/classes/com/sun/crypto/provider/CipherCore.java

```diff
@@ -967,7 +966,7 @@ private int checkOutputCapacity(byte[] output, int outputOffset,
 
     private int finalNoPadding(byte[] in, int inOfs, byte[] out, int outOfs,
                                int len)
-        throws IllegalBlockSizeException, ShortBufferException {
+        throws IllegalBlockSizeException {
```

Comment:
Hmm, is ShortBufferException still needed for `prepareInputBuffer(...)` and `fillOutputBuffer(...)`?

Reply:
Not needed by `fillOutputBuffer(..)` but needed by `prepareInputBuffer(...)`.
IJ was right.